docs/articles/roadmap/README.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: faq
+title: Roadmap
+permalink: /roadmap/
+---
+# Roadmap
+
+This roadmap outlines the most important features currently in our plans or in production.
+
+Please vote on the features you want us to implement first (follow the link and leave a reaction on GitHub). This helps us prioritize and deliver what really matters to you.
+
+Feature                                                                                           | Status
+------------------------------------------------------------------------------------------------- | -------------------
+[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Cucumber Integration](https://github.com/DevExpress/testcafe/issues/2835)                        | Under Consideration
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | Under Consideration
+[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | Under Consideration
+[Custom Actions](https://github.com/DevExpress/testcafe/issues/1535)                              | Under Consideration
+[Custom TypeScript Compilation Options](https://github.com/DevExpress/testcafe/issues/1845)       | Under Consideration
+[Conditional Test Skip](https://github.com/DevExpress/testcafe/issues/1626)                       | Under Consideration
+[Browser Information Available in Test Code](https://github.com/DevExpress/testcafe/issues/481)   | Under Consideration
+
+> We make no guarantee that we implement all the features listed here. Our decisions depend on your feedback.
```

Comment:
Do we really need this bureaucratic disclaimer?

Reply:
DevExpress Roadmap has the similar [disclaimer](
https://community.devexpress.com/blogs/ctodx/archive/2018/03/08/devexpress-roadmap-for-2018.aspx).